ansible/playbooks/create-ansible-tower.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+#
+# sets up the host that runs ansible.nodejs.org
+#
+
+- hosts: infra-softlayer-ubuntu1804-x64-2
```

Comment:
To be safe, can you explicitly exclude this host in https://github.com/nodejs/build/blob/master/ansible/playbooks/jenkins/worker/create.yml ?